index.js

```diff
@@ -18,10 +18,13 @@ import APIError from './helpers/APIError';
 // Load .env
 config();
 
-(async () => {
-  // register scraper
+export const init = (scraperConfig) => {
+  global.scraperConfig = scraperConfig;
```

Comment:
should have config validation before setting it to global. throw error if any.

Reply:
should verify `scraper.json` file here too. any error appears, stop the scraper immediately.